docs/source/udf/profiling.rst

```diff
@@ -42,6 +42,19 @@ To use the :class:`~libertem.executor.inline.InlineJobExecutor`, pass it to the
 
 Then, you can continue as usual, loading data, executing your UDF, etc.
 
+Using progress bar
+---------------------
+
+A progress bar is a graphical tool that shows a far a process has progressed.  It can be used to assess the partitioning of data, since the progress is indicated on a per partition basis in the following way:
+
+.. code-block:: text
+     
+    res = ctx.run_udf(udf=fit_udf, dataset=ds, roi=roi, progress=True)
+    > 100%|██████████| 18/18 [01:17<00:00,  4.29s/it]
+
+This run is using 18 partitions, and took 4.29s/it per partition. 
+Flamescope and Perf can be used for low-level profiling.
```

Comment:
This last part about "flamescope and perf" does not fit into the "progress bar" topic. If there is no real content, i.e. detailed manual for using perf et al, it is better to leave this out first.